server/app/reset.go

```diff
@@ -0,0 +1,19 @@
+package app
+
+import "github.com/mattermost/focalboard/server/services/store"
+
+func (a *App) Reset(c store.Container) error {
+	if err := a.store.CleanUpSessions(0); err != nil {
+		return err
+	}
+
+	if err := a.store.DeleteAllBlocksPermanently(c); err != nil {
+		return err
+	}
+
+	if err := a.store.DeleteAllUsers(); err != nil {
+		return err
+	}
```

Comment:
This is only enabled if corresponding flag in server config file is set to `true`. Also initially there are no any registered users, so it is not clear how to select "specific user".

In general I agree that such testing API for server looks dangerous if it will be enabled in any production setup. So may be we can do better than just a flag in the config.

Also I was thinking about adding log message about enabled testing API and some unit test for server that will check that it is not enabled by default.